js/packages/teams-ai/src/types/ClientCitation.ts

```diff
@@ -44,22 +68,30 @@ export interface ClientCitation {
         url?: string;
 
         /**
-         * Content of the citation. Must be clipped if longer than 480 characters.
+         * Content of the citation. (max length 160)
+         * Must be clipped if longer than 480 characters.
          */
         abstract: string;
 
         /**
          * Used for icon; for now it is ignored.
          */
-        encodingFormat?: 'text/html';
+        encodingFormat?: 'application/vnd.microsoft.card.adaptive';
```

Comment:
This value should only be set if `text` is set otherwise, it should be left undefined